src/wordcloud2.js

```diff
@@ -81,7 +81,9 @@ if (!window.clearImmediate) {
     // "clearZeroTimeout" is implement on the previous block ||
     // fallback
     function clearImmediateFallback (timer) {
-      window.clearTimeout(timer)
+      for (const timerId in timer) {
+        window.clearTimeout(timer[timerId])
+      }
```

Comment:
Revert this change. The `timer` passed in is already the ID.

Reply:
A good catch, reverted.